ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.10.yaml

```diff
@@ -0,0 +1,334 @@
+base_images:
+  base:
+    name: "4.10"
+    namespace: ocp
+    tag: base
+  cli:
+    name: "4.10"
+    namespace: ocp
+    tag: cli
+  openshift_knative-v1.0.0_knative-eventing-kafka-src:
+    name: knative-v1.0.0
+    namespace: openshift
+    tag: knative-eventing-kafka-src
+  openshift_knative-v1.0.0_knative-eventing-src:
+    name: knative-v1.0.0
+    namespace: openshift
+    tag: knative-eventing-src
```

Comment:
We need also `eventing-kafka-broker`
https://github.com/openshift/release/blob/22bf036785f09e86ac3d89dfed6528972f9bee8c/ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.9.yaml#L22-L25

In addition, we're using v1.1 for eventing components:
https://github.com/openshift/release/blob/22bf036785f09e86ac3d89dfed6528972f9bee8c/ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.9.yaml#L14-L25